Drivers/Src/LTC681x.c

```diff
@@ -956,7 +955,8 @@ int8_t LTC681x_rdstat(uint8_t reg, //Determines which Stat  register is read bac
       }
       else if (reg == 2)
       {
-        parsed_stat = data[data_counter++] + (data[data_counter++]<<8);              //Each gpio codes is received as two bytes and is combined to
```

Comment:
Looks like there was some interesting code in here...

Reply:
lol true